index.html

```diff
@@ -2,262 +2,236 @@
 <html lang="en" class="h-100">
 
 <head>
-    <meta charset="utf-8"/>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
-    <link rel="shortcut icon" type="image/png" href=”/img/logo.png”/>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="img/logo.png">
```

Comment:
see https://www.w3schools.com/tags/att_link_sizes.asp, but yeah why was there shortcut?

Reply:
I'm not sure, I def gotta change some VS code settings
Thank you for bringing my screwed up fomatting to light